src/Common/CompileOptions.cs

```diff
@@ -1,5 +1,10 @@
 // Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System.Reflection;
+
+#if MICROSOFT_ENABLE_TELEMETRY
+      [assembly: AssemblyMetadata("OfficialBuilder", "Microsoft")]
```

Comment:
I was thinking of something like this, but no need to change it if you're happy with the OfficialBuilder metadata.

```c#
      [assembly: AssemblyMetadata("TelemetryOptOutDefault", CompileOptions.TelemetryOptOutDefault)]
```